src/io/export_pov.cc

```diff
@@ -59,21 +59,21 @@ void export_pov(const std::shared_ptr<const Geometry>& geom, std::ostream& outpu
       output << "<" << x << ", " << y << ", " << z << ">";
     }
     output << ", <" << ps->vertices[polygon[0]].x() << ", " << ps->vertices[polygon[0]].y() << ", " << ps->vertices[polygon[0]].z() << ">";
-    float r = 1.0, g = 1.0, b = 1.0, a = 0.0;
+    float r = 0xf9 / 255., g = 0xd7 / 255., b = 0x2c / 255., f = 0.;  // CGAL_FACE_FRONT_COLOR
     if (has_color) {
       auto color_index = ps->color_indices[polygon_index];
       if (color_index >= 0) {
         auto color = ps->colors[color_index];
-        r = float(color[0]);
-        g = float(color[1]);
-        b = float(color[2]);
-        a = 1.0 - color[3];
+        r = color[0];
+        g = color[1];
+        b = color[2];
+        f = 1.0 - color[3];
       }
     }
-    if (r == 0 && g == 0 && b == 0)  // work around for black objects in 5185
+    if (r == 0 && g == 0 && b == 0)  // work around for black objects
       g = 1.0;
```

Comment:
I'd say black objects are useful if you want an object that absorbs light with some light object in the background. Ideally we'd support real materials, but changing black to green feels like the wrong idea. If black is really universally unwanted in PovRay, a warning makes sense.